src/gluonts/dataset/repository/datasets.py

```diff
@@ -209,6 +210,12 @@ def get_download_path() -> Path:
             prediction_length=28,
             m5_file_path=get_download_path() / "m5",
         ),
+        "uber_daily": partial(
```

Comment:
Can we name these `uber_tlc_daily` and `uber_tlc_hourly`? That makes it a bit more explicit what data we are using.